helpdesk_mgmt_assign_method/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2025 - TODAY, Kaynnan Lemes <kaynnan.lemes@escodoo.com.br>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class HelpdeskTicket(models.Model):
+    _inherit = "helpdesk.ticket"
+
+    @api.model
+    def default_get(self, fields_list):
+        """Assign user based on team when creating a new ticket."""
+        res = super().default_get(fields_list)
+        team_id = res.get("team_id")
+        if team_id:
+            team = self.env["helpdesk.ticket.team"].browse(team_id)
+            user_id = team.get_new_user().id if team.get_new_user() else False
+            if (not fields_list or "user_id" in fields_list) and "user_id" not in res:
+                res["user_id"] = user_id
+        return res
+
+    def _get_team_user_vals(self, team):
+        """Return dict with user_id from team assignment logic."""
+        return {
+            "user_id": team.get_new_user().id if team.get_new_user() else False,
```

Comment:
```suggestion
            "user_id": team.get_new_user().id
```

Same suggestion as before
Don't call it twice. Once is enough.
Ths method actually returns an empty `res.users` recorset, so reading `id` on it will just return `False`


TBH, though, I see no value for this `_get_team_user_vals` method